flashcards/backend/comments/admin.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib import admin
+
+from comments.models import Comment
+
+
+class CommentAdminConfig(admin.ModelAdmin):
+    """Class **CommentAdminConfig** displays comments list in admin panel
+    based on :class:`comments.models.Comment` model."""
+
+    list_display = (
+        "user",
+        "flashcard",
+        "deck",
+        "description",
+        "created_at",
+    )
+    search_fields = (
+        "user",
```

Comment:
I think you mean to put a look up here. Like for e.g. `deck__name` so it searches foreign key by name. Since flashcard isn't really a table now and does not have a name I would leave an annotation that thos lookup field needs to be added in future, because if you leave it now it will throw this error:

```
FieldError at /admin/comments/comment/
Related Field got invalid lookup: icontains
```